AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -559,23 +569,31 @@ am_Error_e CAmSocketHandler::addTimer(const timespec & timeouts, std::function<v
         return err;
     }
 
-    static auto actionPoll = [](const pollfd pollfd, const sh_pollHandle_t handle, void* userData)
+    auto actionPoll = [this](const pollfd pollfd, const sh_pollHandle_t handle, void* userData)
```

Comment:
Interesting.  Let me know if this is a known C++ best practice compare to the static function.  (I'm not saying it's wrong - just wanting to learn why this is a better approach...)
--> OK